src/main/java/frc/robot/Robot.java

```diff
@@ -177,7 +183,14 @@ public void autonomousInit() {
    * This function is called periodically during autonomous.
    */
   @Override
-  public void autonomousPeriodic() {
+  public void autonomousPeriodic() {// if the selected auto changed, schedule it
+    AutoCommandInterface autoCommandInterface = chosenAuto.getSelected();
+    if (autoCommandInterface != null && autoCommandInterface != m_autonomousCommand) {
+      m_autonomousCommand.cancel();
+      m_robotContainer.robotDrive.setPose(autoCommandInterface.getInitialPose());
```

Comment:
I don't think any of this belongs in autoPeriodic().

Reply:
Concur. There is no way to change the command in Autonomous mode.